src/components/Link/Link-v2.tsx

```diff
@@ -0,0 +1,96 @@
+import clsx from 'clsx';
+import React, { forwardRef } from 'react';
+import type { Size } from '../../util/variant-types';
+import type { IconName } from '../Icon';
+import Icon from '../Icon';
+
+import styles from './Link-v2.module.css';
+
+type LinkHTMLElementProps = Omit<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  'disabled'
```

Comment:
What's omitting `disabled` all about? I thought `disabled` already wasn't an attribute on anchor elements?

Reply:
Good catch, this is probably a carry-over from the `ClickableStyles` stuff that wasn't removed a while back.